docs/getting-started/install.md

```diff
@@ -1,39 +1,48 @@
 ---
-title: Quickstart
+title: Install Infra
```

Comment:
Should we make this title more specific, maybe `Deploying Infra`? I think a very common scenario when opening the docs is a user looking to install the CLI. This makes it look like an end-user has to install an entire Infra server based on the "install infra" title.